.github/workflows/publish_docker_images.yml

```diff
@@ -0,0 +1,45 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+  pull_request:
```

Comment:
maybe we could omit that, since we do want to push a new image on two cases:
1) a push on master
2) a tag

what do you think?

Reply:
Yes, it could be good enough at the moment 👍